src/elements/pages/rvt-user-dashboard.ts

```diff
@@ -65,14 +66,33 @@ export default class RvtUserDashboard extends LitElement {
 				() => html`
 					<div class="text-center p-4 pb-8 self-center">
 						<e-svg class="w-16 h-auto" src="logo/logo-gradient-white" preserve></e-svg>
-						<h1 class="text-5xl mt-2 mb-8">Welcome to Rivet!</h1>
-						<p class="text-lg">Rivet is still in private beta.</p>
-						<p class="my-2 text-lg">Join the waitlist to get early access.</p>
-						<div class="flex justify-center my-8">
-							<stylized-button href="https://b8v8449klvp.typeform.com/rivet" target="_blank">
-								Beta Access Form
-							</stylized-button>
-						</div>
+
+						${when(
```

Comment:
Nit: would this be more legible in a separate function?

Reply:
https://github.com/rivet-gg/hub/pull/28/commits/2fa590237f74d5295cf8bb53241c005e75e5f2dd